### DISCUSSION TITLE
Emergency visits should be hospitals only

### DISCUSSION THREAD

**Discussion by rjake** [synthetichealth/synthea#1282] [Q&A]
The organizations for `encounterclass = 'emergency'` list home care and rehabilitation centers rather than the short list of hospitals that have emergency rooms in the location I requested. In [providers/hospitals.csv](../blob/HEAD/src/main/resources/providers/hospitals.csv) I see a boolean emergency field. Can this be used when identifying ED visits?

The code I ran was 

```r
java -jar synthea-with-dependencies.jar -c resources/synthea.properties -m injuries -p 20 Pennsylvania Philadelphia
```
and then this is the SQL code I used

```sql
select
    og.name,
    count(*) as n
from
    main.encounters                en
    left join main.organizations   og on og.id = en.organization
where
    en.encounterclass = 'emergency'
group by 1
```
In this example, none of these in blue have emergency rooms

![image](https://user-images.githubusercontent.com/14003618/233395903-702d1e92-63e2-4736-a391-ba86ba1568ed.png)

Am I doing something wrong or is this something I should make an issue for?

**Comment by jawalonoski** [comment]
## Emergency Rooms

The boolean `emergency` field is what is being used to determine facilities that have an emergency room.

This is a screenshot of `src/main/resources/providers/primary_care_facilities.csv`:

![image](https://user-images.githubusercontent.com/1054765/233402903-0d563704-7b75-47df-bbd2-cedcc367944d.png)

All of those facilities indicate emergency rooms. The Lafayette-Redeemer entry is similarly flagged in the `nursing.csv` file.

There could be a data quality issue. We compiled this data from multiple public sources (see subset in the table below), primarily the list of publicly available providers from CMS. It is possible that the facility in question shares the provider number or NPI of another facility with an emergency room, and therefore also shares the emergency flag.

| Source | Description |
|--------|-------------|
| Medicare Fee-For-Service Public Provider Enrollment | This data set includes information about organizations and clinicians such as NPI, PECOS number, provider type, city, state, and zip code. |
| Provider of Services Files | This data set includes PRVDR_NUM as well as addresses, FIPS county, emergency status, bed counts, CLIA numbers, and provider category codes. |
| National Plan and Provider Enumeration System (NPPES) | This data set was used for additional addresses and telephone numbers. |
| npi_medicarexw | This data set links NPI and PRVDR_NUM, along with PIN and UPIN numbers. |

## M-Flag

⚠️ See https://github.com/synthetichealth/synthea/wiki/The--M-Feature

**Comment by TravisHaussler** [comment]
I am also running into undesired data here while inspecting pregnancy data - patients giving birth in hospice care, etc... my goal is also to just use a list of hospital.csv overrides I've created.

For now I am manually deleting the emergency field in the providers source data in the jar, but that is not an elegant solution
